src/classes/RD2_OpportunityService.cls

```diff
@@ -141,15 +143,19 @@ public with sharing class RD2_OpportunityService {
 
 
     /***
-    * @description Indicates if the Opportunity already exists
+    * @description Indicates if the Opportunity should be created
```

Comment:
Can you update the description here to explain why it's checking for a specific value in the `Recurring_Donation_Installment_Number__c` field? If anyone else (or even any of us) look at this code in 6 months we're not going to remember why this is being done and what the business use case was.